docs/e2e-test-guide.md

```diff
@@ -0,0 +1,164 @@
+# Argo CD Operator E2E Test Guide
+
+E2E tests are written using [KUTTL](https://kuttl.dev/docs/#install-kuttl-cli).
+
+## Requirements
```

Comment:
Should probably mention that  the redis ha tests require a cluster that has at least three worker nodes.

Reply:
Ack. Also added this a per-requisite for running the tests.